microprofile/testing/junit5/src/main/java/io/helidon/microprofile/testing/junit5/AddConfigs.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.helidon.microprofile.tests.testng;
+package io.helidon.microprofile.testing.junit5;
```

Comment:
Why move testng ones to junit and vice-versa?

Reply:
I did not. I refactored it using IDE by moving the correct ones around

Reply:
Fixing

Reply:
I have used `git mv` to move the files, yet still some of them appear to be incorrectly marked. I would prefer not to manually change the git patch, so I have no choice but to leave it as is